src/common/include/gudhi/random_point_generators.h

```diff
@@ -26,6 +27,8 @@
 
 namespace Gudhi {
 
+constexpr double pi = boost::math::constants::pi<double>();
```

Comment:
If this file had its own namespace, that would be ok, but here in namespace Gudhi, it means that if we do the same in another file, it will fail. Vincent was suggesting to create a separate header for that, but I'd rather just add `using namespace boost::math::double_constants;` in each function that uses `pi`. Note that with `double_constants` you can keep using non-template `pi` which has type `double`.
(the genericity in that file is a bit fake, it kind of assumes that FT is `double`, so no need to try `constants::pi<FT>` for now)